ansible/roles/debops.prosody/meta/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+dependencies:
+  - role: debops.ferm
```

Comment:
This dependency is defined in the `prosody.yml` playbook and shouldn't be defined here, since the `debops.ferm` role will be executed twice in this case. Please remove it and set `dependencies: []`.